zend_abstract_interface/headers/tests/php7-8/headers.cc

```diff
@@ -0,0 +1,45 @@
+extern "C" {
+#include "zai_sapi/zai_sapi.h"
+#include "headers/headers.h"
+
+#include <Zend/zend_API.h>
+}
+
+#include <catch2/catch.hpp>
+#include <cstring>
+
+#define TEST(name, code) TEST_CASE(name, "[zai headers]") { \
+        REQUIRE(zai_sapi_spinup()); \
+        ZAI_SAPI_TSRMLS_FETCH(); \
+        ZAI_SAPI_ABORT_ON_BAILOUT_OPEN() \
+        { code } \
+        ZAI_SAPI_ABORT_ON_BAILOUT_CLOSE() \
+        zai_sapi_spindown(); \
+        register_custom_server_variables = NULL; \
+    }
+
+static void define_server_value(zval *arr) {
+    add_assoc_string(arr, "HTTP_MY_HEADER", "DataDog");
```

Comment:
Tiny nit: the second "d" is lowercase: `Datadog`. 🐶

Reply:
The name of the github org is with two captial D's … so …

Reply:
That's a relic from an older time that github doesn't currently let you fix! Or at least that's what I've heard.